plugin/evm/vm.go

```diff
@@ -1208,6 +1284,40 @@ func (vm *VM) GetSpendableFunds(
 	return inputs, signers, nil
 }
 
+// getSpendableFundsEVMInputForAddress returns an EVMInput to total [amount] of [assetID]
+// owned by [address]. If the balance of the address is zero, returns nil, nil.
+func (vm *VM) getSpendableFundsEVMInputForAddress(
```

Comment:
Refactor contents of `GetSpendableFunds` loop so that this method can accept a `common.Address` instead of a `*crypto.PrivateKeySECP256K1R`